service/mantle/fedex/FedexServices.xml

```diff
@@ -0,0 +1,162 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<services xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+          xsi:noNamespaceSchemaLocation="http://moqui.org/xsd/service-definition-2.1.xsd">
+  <service verb="request" noun="ShippingLabels">
+        <implements service="mantle.shipment.CarrierServices.request#ShippingLabels"/>
+        <out-parameters>
+            <parameter name="responseMap"/>
+            <parameter name="labelRequestMap"/>
+        </out-parameters>
+        <actions>
+
+            <!-- Fetching Shipment Entity -->
+            <entity-find-one entity-name="mantle.shipment.Shipment" value-field="shipment" for-update="true"/>
+            <if condition="!shipment"><return error="true" message="Shipment not found with ID ${shipmentId}"/></if>
+
+            <!-- Fetching ShipmentRouteSegment Entity -->
+            <entity-find-one entity-name="mantle.shipment.ShipmentRouteSegment" value-field="routeSegment"/>
+            <if condition="!routeSegment">
+                <return error="true" message="Shipment Route Segment not found with ID ${shipmentId}:${shipmentRouteSegmentSeqId}"/>
+            </if>
+
+            <!--If shipmentPackageSeqId present, will call service for one package, else for all packages-->
+            <entity-find entity-name="mantle.shipment.ShipmentPackageRouteSeg" list="packageRouteSegList">
+                <econdition field-name="shipmentId"/><econdition field-name="shipmentRouteSegmentSeqId"/>
+                <econdition field-name="shipmentPackageSeqId" ignore-if-empty="true"/>
+            </entity-find>
+
+            <!-- Calling LabelRequestMap for all Packages -->
+            <service-call name="mantle.fedex.FedexServices.make#LabelRequestMap" in-map="context" out-map="request"/>
+            <if condition="!request.labelRequestMap"><return/></if>
+            <set field="labelRequestMap" from="request.labelRequestMap"/>
+
+            <!-- Setting Connection With Fedex -->
+            <service-call name="mantle.fedex.FedexServices.send#FedexRequest" out-map="response"
+                in-map="[shippingGatewayConfigId:shippingGatewayConfigId, method:'POST', path:'ship/v1/shipments',
+                contentType:'application/json', requestMap:request.labelRequestMap, xLocale:'en-US']"/>
```

Comment:
why is xLocale used here?

Reply:
xLocale indicates the combination of language code and country code. Eg: en_US. But it is not a required parameter in the requestMap.

Reply:
Okay, but does your generic service send#FedexRequest accepts this parameter?

Reply:
Mam x-Locale comes in the header parameter of POST request, so you are right, it should be removed from here.